.github/workflows/add-prs-to-project.yml

```diff
@@ -14,11 +19,25 @@ jobs:
       TEAM_LABEL: 'team-wallet-framework'
 
     steps:
+      - name: Get existing PRs
+        if: github.event.inputs.scan_existing_prs == 'true'
+        id: get-prs
```

Comment:
Where is `get-prs` being used? It's not referenced in this workflow so I'm a little confused 🤔

Reply:
The inputs area above in `workflow_dispatch` should allow me to run this manually in the github actions view and select "true" the input field. That will trigger `get-prs` to run. 

https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

I have not tested this on a fork but given it should only impact a manual run and shouldn't cause any harm I was hoping to be able to test it once merged.

Reply:
Oh sorry — I meant that I see how this step could get run, and I see that this step reads a list of pull requests, but I don't see where those pull requests are getting used. Does the `add-to-project` implicitly use the results of this step somehow?